code/modules/mob/living/living.dm

```diff
@@ -1124,7 +1124,7 @@
 	var/knockdown = IsKnockdown()
 	var/ignore_legs = get_leg_ignore()
 	var/in_stasis = IsInStasis()
-	var/canmove = !IsImmobilized() && !stun && conscious && !paralyzed && !buckled && (!stat_softcrit || !pulledby) && !chokehold && !IsFrozen() && !in_stasis && (has_arms || ignore_legs || has_legs)
+	var/canmove = !IsImmobilized() && !stun && conscious && !paralyzed && !buckled && (!stat_softcrit || !pulledby) && !chokehold && !IsFrozen() && !in_stasis
```

Comment:
This feels like it's going to change some things in a weird way, but I can't immediately say what

Reply:
I think the weirdness is intentional: it lets nuggets crawl around now.  I thought carefully and couldn't come up with anything else it would affect offhand.

Reply:
check wheelchair movement to make sure it doesn't just break them. Don't think it will but thats the only thing i can think of